### PR TITLE
Styles

### PR DESCRIPTION
**Pull request by JohnFTitor**
# Hi there 👋🏻 

Thanks for taking the time to review the project. Here are some things important to note about it:

## Sass

The project was built using Sass. The only difference was that I needed to add an additional loader to webpack, but the result is overall the same (A CSS output). Additionally, I used ```MiniCssExtractPlugin``` to generate an external CSS file in the dist folder instead of the styles in the head of the document. It has the same dynamic response, it's just that the output looks more organized.

### Changes implemented in "styles" branch 

- ✅  Add Styles to elements inside the page.
- ✅  Add animations to buttons.
- ✅  Add sorting to leader board elements.

## Final notes

Is expected that this project meets the requirements. Please let me know if anything is left to be done or can be improved. Again, thanks for the review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
+How much did you score last time you played with friends? What about having a healthy but fun competition? Keep track of your points with the LeaderBoard!.
 
 ## Built With
 
-Tecnologies used
+- HTML/SCSS, JavaScript
+- WebPack, Jest
+- Visual Studio Code
 
 ## Live Demo
```
Comment (ElijahTobs) [inline README.md:18]: - [x] The live demo link you provided is broken. Kindly add the right link

File: README.md [hunk 2]
```diff
@@ -1,16 +1,18 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Title
+# Leader Board: Hit the API!
 
-> Project headline description
+> This simple project aims to connect to an API to fetch score data and display it on a LeaderBoard box, as well as provide the tool to submit a new score.
 
 ![Preview](./preview.gif))
```
Comment (ElijahTobs) [inline README.md:8]: - [ ] This preview does not correspond with the output of your project. When I try to submit a score, the leaderboard doesn't update. The style is also not the same as the wireframe.

image

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required ⚠️ 🚦
## Hi @JohnFTitor 👋

You've done really well so far! 👍 

## Major Highlights 💯 

You've followed the Github flow rules. ✔️ 
Good commit messages ✔️ 
No linter errors ✔️ 

_However, there are some changes that you are required to make before this project can be approved._

## Required Changes ♻️


_See the inline comments below_


_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

#### "Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
## Happy Coding :computer: 

_Reviewed by_  **Elijah O. Ayandokun**

File: README.md (hunk 2 above)
Comment (JohnFTitor) [reply README.md:8]: Hi @ElijahTobs. 

It's strange you get that result. It looks like webpack is not bundling the styles or the functionality. Is it perhaps an issue with webpack? Here it works as shown in the preview.

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢

### Hi @JohnFTitor  👋🏻, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**